proposals/PROPOSALS.md

```diff
@@ -0,0 +1,142 @@
+
+If you’re motivated enough to go through this involved but rewarding process, go on with writing and keep on reading.
+
+## What's the process for submitting a HIP?
+
+There are four major steps in the HIP process:
+
+1. Initial informal discussion (2 weeks)
+2. Submission
+3. Formal presentation (up to 1 month)
+4. Formal evaluation (up to 2 months)
+
+### Initial informal discussion (2 weeks)
+
+Before submitting a HIP, it is required that you perform necessary preparations:
+
+- Discuss your idea on the [Haskell.org Discourse](https://discourse.haskell.org/). Currently, we suggest cross-posting on the Haskell Foundation Slack for higher volume commentary. Create a Discourse topic under the category "Haskell Foundation", that starts with "Pre-HIP” and briefly describe what you would like to change and why you think it’s a good idea.
+
+- Proposing your ideas on the Discourse is not an optional step. For every change to the ecosystem, it is important to engage in Due Diligence with the community and relevant stakeholders. Use this step to promote your idea and gather early feedback on your Pre-HIP proposal. It may happen that experts and community members may have tried something similar in the past and may offer valuable advice.
+
+
+### Submission
+
+A HIP is a Markdown document written in conformance with the [process template](https://github.com/haskellfoundation/tech-proposals/blob/main/TEMPLATE.md). When such changes significantly alter an existing library or tool, the author is invited to provide a proof of concept. Delivering a basic implementation can speed up the process dramatically. If your changes are big or somewhat controversial, don’t let people hypothesize about them and show results upfront.
+
+A HIP is submitted as a pull request against [the official Haskell tech proposal
+repo](https://github.com/haskellfoundation/tech-proposals). Within a week of receiving the pull request, members of the HF Technical Track will acknowledge your submission, validate that it conforms to the proposal template guidelines (see: [TEMPLATE.md](TEMPLATE.md)) and provide feedback to improve the overall quality of the document (if necessary). When the document conforms to the template guidelines, it is ready for formal evaluation by the HF Technical Track members, as well as the broader community. 
+
+### Formal evaluation (up to 5 iterations)
+
+While the majority of general technical commentary and feedback will occur in the proposals repo, it's hard to tell when a particular proposal is finalized. This what we call "Formal Evaluation". 
+
+Formal Evaluation of a proposal is done in iterations. The maximum number of iterations is five. These iterations take place in the HF Tech Track meetings and are usually monthly. However, they can last longer, in which case the author has more time to implement all the required changes. 
```

Comment:
HF tech track meetings are *weekly*, however, we may want to add a meeting series specifically for HIP review. The language will change as we decide what to do here.